tests/client/amo/components/TestAddonDetail.js

```diff
@@ -139,6 +139,23 @@ describe('AddonDetail', () => {
                    'About this extension');
   });
 
+  it('uses the summary as the description if no description exists', () => {
+    const addon = { ...fakeAddon };
+    delete addon.description;
+    const rootNode = renderAsDOMNode({ addon });
+    assert.equal(
+      rootNode.querySelector('.AddonDescription-contents').textContent,
+      addon.summary);
+  });
+
+  it('uses the summary as the description if no description exists', () => {
+    const addon = { ...fakeAddon, description: '' };
```

Comment:
For cases like this I think it's helpful to make an explicit dependency on the `summary` property, like this:
````js
const addon = { ...fakeAddon, description: '', summary: 'short text' };
...
assert.equal(
  rootNode.querySelector('.AddonDescription-contents').textContent,
  'short text');
````